src/core/issues-notes.js

```diff
@@ -203,6 +193,30 @@ function createIssueSummaryEntry(l10nIssue, report, id) {
   `;
 }
 
+/**
+ *
+ * @param {*} issueList
+ */
+function makeIssueSectionSummary(issueList) {
+  const issueSummaryElement = document.getElementById("issue-summary");
+
+  if (!issueSummaryElement) return;
+  const heading = issueSummaryElement.querySelectorAll("h2, h3, h4, h5, h6");
+  const defaultHeading = hyperHTML`<h2>${l10n.issue_summary}</h2>`;
+
+  issueList.hasChildNodes()
+    ? issueSummaryElement.append(issueList)
+    : issueSummaryElement.append(
+        hyperHTML`<p>There are no issues listed in this specification.</p>`
```

Comment:
We should localize this string. Please add it to `localizationStrings` above as "no_issues_in_spec". 

```suggestion
        hyperHTML`<p>${l10n.no_issues_in_spec}</p>`
```

For Spanish, please use: 

"No hay problemas enumerados en esta especificación." 

@thijsbrentjens, could you suggest a Dutch translation for: 

"There are no issues listed in this specification."

Reply:
I did a google translate for this. Let me know if that needs some changes.